src/app/f2f/controllers/landingPage.js

```diff
@@ -12,6 +12,10 @@ class LandingPageController extends BaseController {
         // Show thin file user screen
         req.sessionModel.set("isThinFileUser", true);
       }
+      if (configData && configData.pcl_enabled) {
```

Comment:
Does this condition make sense if configData.pcl_enabled is set to false? 
Do we not want something more like this